.github/workflows/pr-validation.yml

```diff
@@ -121,6 +124,9 @@ jobs:
         
       - name: Build docker image & build configuration YAML
         run: |
+          df -H
+          docker info
+          docker image list
```

Comment:
Do we still need these logs here to be merged?

Reply:
For now, it is probably good to keep these. Maybe we remove them later.

Reply:
In that case, I'd suggest leaving a TODO here so we know this is just test stuff and can be removed later.

Reply:
I've removed them now